vault_core/src/vault.rs

```diff
@@ -158,6 +164,7 @@ impl Vault {
         deposit_fee_bps: u16,
         withdrawal_fee_bps: u16,
         reward_fee_bps: u16,
+        program_fee_bps: u16,
```

Comment:
nit: this should not be an input - and it should only be set by the Config.program_fee_bps.

set it to 0 or u64:MAX for uninitialized

Reply:
discussed offline - renaming constructor arg but leaving here